public/Lab-Files/Lesson03/Challenges/box-positioning.css

```diff
@@ -0,0 +1,13 @@
+body {
+  box-sizing: border-box;
+}
+.row1 div,
+.row2 div {
+  padding:10px;display:inline-block;border:1px solid black;width:50px;height:50px;margin:10px;font-weight:bold;
+}
```

Comment:
Good job on completing the challenges. You were very creative on this one.

I'd suggest that when you write your CSS you give each property its own line. I think we went over it in class.

```css
.row2 div {
     padding: 10px;
     display: inline-block;
     border: 1px solid black;
}
```